data-model/src/main/java/org/hypertrace/core/datamodel/shared/trace/StructuredTraceBuilder.java

```diff
@@ -365,11 +365,21 @@ private void processEventRefList(ByteBuffer traceId, Event event) {
         //TODO: consider creating an empty event node?
         continue;
       }
-      //todo: Read about what can we do with FOLLOWS_FROM
-      //Ignore FOLLOWS_FROM for now.
-      if (!eventRef.getRefType().equals(EventRefType.CHILD_OF)) {
-        continue;
-      }
+
+    /*
+     * For Follow from relationship:
+     * As, both the construct `child_of` and `follow_from` represent parent-child relation in common
+     * where in one case parent is interested in child span's result while in other case not.
+     *
+     * So, to support common behaviour, we will be establish link for `follow_from` as well.
+     * commenting out this part.
+     *
+     *      if (!eventRef.getRefType().equals(EventRefType.CHILD_OF)) {
+     *
+     *   continue;
+     * }
+     * Ref: https://github.com/hypertrace/hypertrace/issues/234.
+     */
```

Comment:
This looks fine as long as eventRefList's size is one. If it's more than one, it will create multiple parents and introduce diamonds in a graph. I think it should be fine. 
Can you also add a note in the ideal scenario, we will have a single parent for the event?

Reply:
```suggestion
     note: Ideally, an event should have a single parent
     */
```